book_source/03_topical_pages/93_installation/03_install_OS/04_Installing-PEcAn-OSX.Rmd

```diff
@@ -86,7 +154,20 @@ make
 sudo make install
 ```
 
-#### Apache Configuration
+#### Install DuckDB
```

Comment:
Looks to me like the {duckdb} R package includes everything with no need to install any libraries separately. https://duckdb.org/docs/installation/?version=stable&environment=r  

...Unless PEcAn is using duckdb through some non-R API?

Reply:
Ah, I just read 

```
ERROR: dependency ‘duckdb’ is not available for package ‘duckdbfs’
* removing ‘/Users/dlebauer/R/x86_64-pc-linux-gnu-library/4.4/duckdbfs’
```

But perhaps that was referring to the R package rather than the DuckDB itself? Looks like it isn't in the r package documentation. I'll remove